wininst/nsiscript.top

```diff
@@ -110,53 +114,53 @@ Section "Core GAP system" SecGAPcore
   # Set output path to the installation directory
   SetOutPath $INSTDIR
   # Put files there
-  File gap-4.10.2\*.*
+  File $GAP_DIR\*.*
 
   SetOutPath $INSTDIR\.libs
-  File /r gap-4.10.2\.libs\*.*
+  File /r $GAP_DIR\.libs\*.*
 
   SetOutPath $INSTDIR\autom4te.cache
-  File /r gap-4.10.2\autom4te.cache\*.*
+  File /r $GAP_DIR\autom4te.cache\*.*
```

Comment:
BTW, does this mean that the dir `autom4te.cache` is copied over? If so: why do we do that? It's only need for compiling GAP, and not anymore afterwards.

Same holds for `gen` and `obj`  and probably also `.libs` (that should be verified on Windows, though, as in: if GAP starts without it, then it can go).